### ISSUE TITLE
Does not match m.youtube.com

### ISSUE THREAD

**Issue by SandroJijavadze** [boomzillawtf/nodebb-plugin-youtube-embed#15]
Does not match m.youtube.com

Will try to make changes myself once I have time to figure out how this plugin works (New to nodejs world).

**Comment by mberglundmx** [comment]
I see the same problem, have you been able to create a fix?

**Comment by SandroJijavadze** [comment]
@mberglundmx 
No, and I no longer have a nodebb community to check this but it seems like you only need to modify this regex: 
https://github.com/boomzillawtf/nodebb-plugin-youtube-embed/blob/master/library.js#L14
